### DISCUSSION TITLE
End metadata printout with a new line

### DISCUSSION THREAD

**Discussion by bitdivine** [dfinity/sdk#3210] [Feature Requests]
Canister metadata output can be hard to read:

```
max@sinkpad:~/dfn/nns-dapp $ dfx canister metadata --network ic xnjld-hqaaa-aaaal-qb56r-cai git_commit_id
b8bb1ca9bdb6d40d2b45d36fafc515acef859aabmax@sinkpad:~/dfn/nns-dapp $ 
```
Yes, the output is mixed together with the next terminal prompt.

It would be nice if `dfx canister metadata` could make sure to finish the output with a newline, either always adding one or adding one when the metadata itself doesn't end with a newline.

What do you think?

**Comment by ghost** [comment]
We need to take care not to add anything to what the other end of a pipe would see, or when writing to a file.
